recipe/meta.yaml

```diff
@@ -57,7 +56,7 @@ test:
 
 about:
   home: https://symbolixau.github.io/spatialwidget/articles/spatialwidget.html
-  license: GPL-3
+  license: GPL-3.0-or-later
```

Comment:
meta data tells (see commented out at the end of the meta.yaml) that this should be `GPL-3.0`.

Reply:
This is the SPDX form for the license as mentioned by the linter. GPL3 is not sufficient for SPDX.

Reply:
```suggestion
  license: GPL-3.0
```

Reply:
Adjusted but the linter is not happy